server/validation/is-empty.js

```diff
@@ -0,0 +1,7 @@
+const isEmpty = value => (
+  value === undefined
+      || value === null
+      || (typeof value === 'object' && Object.keys(value).length === 0)
+      || (typeof value === 'string' && value.trim().length === 0)
+);
```

Comment:
What about a test for an empty array. We might not need it but I think it is good for completeness.